src/QsCompiler/CompilationManager/AssemblyLoader.cs

```diff
@@ -48,11 +48,17 @@ public static bool LoadReferencedAssembly(Uri asm, out References.Headers header
             using var assemblyFile = new PEReader(stream);
             if (ignoreDllResources || !FromResource(assemblyFile, out var compilation, onDeserializationException))
             {
+                PerformanceTracking.TaskStart(PerformanceTracking.Task.HeaderAttributesLoading);
                 var attributes = LoadHeaderAttributes(assemblyFile);
+                PerformanceTracking.TaskEnd(PerformanceTracking.Task.HeaderAttributesLoading);
+                PerformanceTracking.TaskStart(PerformanceTracking.Task.ReferenceHeadersCreation);
                 headers = new References.Headers(id, attributes);
+                PerformanceTracking.TaskEnd(PerformanceTracking.Task.ReferenceHeadersCreation);
                 return ignoreDllResources || !attributes.Any(); // just means we have no references
             }
+            PerformanceTracking.TaskStart(PerformanceTracking.Task.ReferenceHeadersCreation);
```

Comment:
This task has the same name as an earlier one, in line 54. If they both get executed, does this task start-end block override the other one?

Reply:
No, they add up. The total accounted time for the task would be the sum of the time spent in both sections.